src/api/tests/service/shifu/test_course_copy.py

```
@@ -0,0 +1,446 @@
+        "flaskr.route.user.validate_user",
+        lambda _app, _token: dummy_user,
+        raising=False,
+    )
+    return dummy_user
+
+
+def _clear_config_caches() -> None:
+    try:
+        config_module.__ENHANCED_CONFIG__._cache.clear()
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)